exercise_02.py

```diff
@@ -0,0 +1,25 @@
+def reverse(number):
+    n_list = list(str(number))
+    number = ""
+    for i in range(1, len(n_list) + 1):
+        number += n_list[-i]
+
+    return number
+
+
+def is_palindrome(number):
```

Comment:
The name of this function suggests you will return a boolean indicating whether or not the passed `number` is a palindrome. However, it is like you like to lie to the user, and you are returning a string. Please, return the boolean we are expecting, and display the messages properly in your main function.